pkg/kubelet/kubelet.go

```diff
@@ -920,10 +926,20 @@ func (kl *Kubelet) listPodsFromDisk() ([]types.UID, error) {
 // Starts garbage collection threads.
 func (kl *Kubelet) StartGarbageCollection() {
```

Comment:
suggestion: can we start garbage collection in kubelet.Run(), so it's the same as other components?